dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -307,8 +310,67 @@ public void findOneTestWrongUUID() throws Exception {
         Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
 
         getClient().perform(get("/api/core/items/" + UUID.randomUUID()))
-                   .andExpect(status().isNotFound());
+                   .andExpect(status().isNotFound())
         ;
+
+    }
+
+    @Test
+    public void patchTest() throws Exception {
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+
+        //2. One public item
+        Item item = ItemBuilder.createItem(context, col1)
+                               .withTitle("Public item 1")
+                               .withIssueDate("2017-10-17")
+                               .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                               .withSubject("ExtraEntry")
+                               .build();
+
+        // withdraw item
+        getClient(token).perform(patch("/api/core/items/" + item.getID())
+            .content("[{\"op\":\"replace\",\"path\":\"withdraw\"}]")
+            .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$.uuid", Matchers.is(item.getID().toString())))
+                        .andExpect(jsonPath("$.withdrawn", Matchers.is(true)))
+                        .andExpect(jsonPath("$.inArchive", Matchers.is(false)));
+
+        // should return 422 because item is not in archive
+        getClient(token).perform(patch("/api/core/items/" + item.getID())
+            .content("[{\"op\":\"replace\",\"path\":\"withdraw\"}]")
+            .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                        .andExpect(status().isUnprocessableEntity());
+
+        // reinstate item
```

Comment:
better create separate test methods for the different scenarios
- withdrawn
- reinstatiate
- private
- unprivate (discoverable)